strawberry/test/client.py

```diff
@@ -0,0 +1,34 @@
+from abc import abstractmethod
+from dataclasses import dataclass
+from typing import Any, Dict, Optional
+
+from typing_extensions import Literal, TypedDict
+
+
+@dataclass
+class Response:
+    errors: Optional[Dict[str, Any]]
+    data: Optional[Dict[str, Any]]
+    extensions: Optional[Dict[str, Any]]
+
+
+class Body(TypedDict, total=False):
+    query: str
+    variables: Optional[Dict[str, Any]]
+
+
+class BaseGraphQLTestClient:
+    def __init__(self, client):
+        self._client = client
+
+    @abstractmethod
+    def query(
+        self,
```

Comment:
`BaseGraphQLTestClient` needs to inherit from `ABC` to use `@abstractmethod` properly.